files/image_config/monit/memory_checker

```diff
@@ -94,6 +96,31 @@ def check_memory_usage(container_name, threshold_value):
         sys.exit(4)
 
 
+def get_running_container_names():
+    """Retrieves names of running containers by talking to the docker daemon.
+
+    Args:
+        None.
+
+    Returns:
+        running_container_names: A list indicates names of running containers.
+    """
+    running_container_names = []
+    try:
+        docker_client = docker.DockerClient(base_url='unix://var/run/docker.sock')
+        container_obj = docker_client.containers
+        container_list = container_obj.list(filters={"status": "running"})
+        for container in container_list:
+            running_container_names.append(container.name)
+    except (docker.errors.APIError, docker.errors.DockerException) as err:
```

Comment:
@yozhao101 i would suggest to rewrite like this:
```python
    try:
        docker_client = docker.DockerClient(base_url='unix://var/run/docker.sock')
        running_container_list = docker_client.containers.list(filters={"status": "running"})
        running_container_names = [ container.name for container in running_container_list ]
    except (docker.errors.APIError, docker.errors.DockerException) as err:
```

Reply:
Updated.